coordinated-entry-screening/static/css/custom.css

```diff
@@ -1,6 +1,66 @@
+:root {
+    --blue: #0178BC;
+    --light-blue: #4da0d0;
+}
```

Comment:
I would choose these colors for either the badges *or* the bar chart; otherwise it appears that the number correlates to a part of the chart.

Reply:
that's a very good point

Reply:
Red screaming "DANGER" to me! How about one of the ones on the left? https://coolors.co/03b5aa-037971-0178bc-023436-00bfb3

Reply:
Ha, I do like it! But yes, something that can be easily changed.